### DISCUSSION TITLE
Quais os sites que vocês acham vagas para FrontEnd?

### DISCUSSION THREAD

**Discussion by marcusvcrd** [frontendbr/forum#920] [Discussão]
Abri esse tópico, devido estar sempre de olho nas vagas de frontend.
Porem até o linkdIn está redirecionando para vários outros sites, no qual tem que fazer login para achar vagas ou enviar currículos. Antigamente gostava bastante do linkdIn.

Um dos poucos sites que não tem essa frescura é o https://comoequetala.com.br/

Então surgiu minha duvida, onde a galera de front olham as vagas?

**Comment by NandoSangenetto** [comment]
Aqui mesmo, mas no repositório de vagas:

https://github.com/frontendbr/vagas

**Comment by renanlara** [comment]
Além do citado acima que é muito bom, tem o [trampo.co](http://trampos.co/).

**Comment by NandoSangenetto** [comment]
Além da minha resposta, tem o Bunee.io do @bernarddeluna: https://bunee.io/devs

**Comment by leandroruel** [comment]
LinkedIn. ultimamente tá dando de 10 a 0 em trampos, Vagas.com, Infojobs (ew!)

**Comment by marcusvcribeiro** [comment]
Porem o LinkdIn está redirecionando para esses sites de vagas quando voce
clica em candidatar-se.
Ja usei até o stackoverflow Jobs.

2018-02-01 16:18 GMT-02:00 Leandro Ribeiro Ruel <notifications@github.com>:

**Comment by vitocaio** [comment]
@marcusvcribeiro mesmo assim a maior concentração de vagas esta no Linkedin

**Comment by bernarddeluna** [comment]
Nao vou falar do meu, pois tentarei focar em comunidade, antes de qq coisa.

Existem diversos job boards por aí, mas a maioria deles não funciona, pois os melhores devs não estão procurando por emprego, por isso precisam ser encontrados (recrutamento ativo).

Linkedin tem uma estatistica pra isso, informando que apenas **25%** dos profissionais qualificados são impactados por uma vaga de emprego.

Então, eu acredito na seguinte ordem para recrutamento de programadores:

- Indicação (sempre a melhor opção)
- Recrutamento ativo (plataformas e consultorias de hunting)
- Recrutamento passivo (job boards, vagas no site)

Então, usando como perspectiva:

**Empresa:** Criar uma cultura de indicações internas. Vagas servem para gerar employer branding, mas não para captação. Por isso, focaria o esforço de captação no recrutamento ativo (daí sim a Bunee.io pode ajudar bastante).

**Programador(a):** Abrir e qualificar o máximo sua rede de contatos no Github e Linkedin. Deixar o Github o mais ativo possível, Escrever artigos em iMasters, Fórums e Tableless, tirar micro certificados em Alura, MXCursos, Udacity, Udemy, etc, estar ativo na comunidade no twitter, estar em eventos e sempre conversar com a comunidade, estar cadastrado em todas as plataformas de recrutamento ativo (Bunee.io e Vulpi) e Rodadas de contratação passiva (revelo, geekhunter). 

Espero ter ajudado pessoal!

**Comment by rubenmarcus** [comment]
Cara antes de fazer um mochilao pela India fui num evento de Front aqui em SP que estava um cara da contratado.me hoje , www.revelo.com.br, E pra mim foi o melhor sistema, Eu lembro que na palestra ele passou a média de salarios de front , e eu também acompanhava no lovemondays, e pela minha experiência e conhecimento eu tava ganhando quase como Júnior, e ja tinha bagagem quase de senior. Então agora em Janeiro voltei da viagem me cadastrei lá e na primeira semana estava indo em entrevistas, e depois de 3 semanas que 6 empresas me procuraram, eu estou empregado, com o salário que gostaria, e aind atem bonificação pela contratação (20%) do salário acordado. Eu me assustei um pouco por ter testes, mas fui bem nos testes, te aconselho a ir no HackerRank,Upwork e sites de teste multipla escolha, e também fazer testes práticos , isso te dá uma confiança gigante na hora da entrevista. Recentemente fiz um pra uma vaga duma empresa conhecida,pensei que tinha ido mó mal, a consultora falou que eu fui um dos melhores, eu vi varias questões de js avançado e lógica, que eu me considero ruim ainda e pensei em desistir ,mas fui lá e fiz, deixei duas em branco mesmo sabendo a resposta,com medo de responder errado, e algumas q eu achava fácil de lógica eu enrolei p kraio pq eu fiz de várias formas e nunca parecia que tava bom.kkkkk     Mas recomendo esse mesmo, no Linkedin se vc por seu perfil bem descritivo com as funções nos projetos q vc fez, tecnologias,link do portfólio link do git, currículo, seu telefone, cara eu tenho uns 3-4 convites por dia de vaga, nossa área tá muito aquecida

**Comment by rubenmarcus** [comment]
na gringa : Glassdoor,Monster,Indeed,Jooble,Upwork,Freelancer,Fivrr,Stack Overflow Jobs, Remote.io, Startup Jobs Asia (recebi uma vaga pra Cingapura kkk deve see louco trabalhar lá ou em Hong Kong)    Sei que os maiores salários tão em Dubai e na Austrália....  Cada país pode ter seus sites especificos tb só procurar

**Comment by letanure** [comment]
http://berlinstartupjobs.com/engineering/
https://www.xing.com/

**Comment by leandroruel** [comment]
gerei essa lista aqui para pessoas que estão procurando vagas em determinadas empresas, com bom salario e beneficios
https://www.lovemondays.com.br/melhores-empresas-por-setor/tecnologia-e-telecomunicacoes/remuneracao-e-beneficios

**Comment by nicholasess** [comment]
Vagas de React - https://github.com/react-brasil/vagas

**Comment by phsantiago** [comment]
Por ordem das que tiveram mais retorno para mim:

* Indicação
* LinkedIn
* frontendbr/vagas
* trampos.co

Existe o programathor também, nunca usei mas parece bom.

**Comment by pedrocss** [comment]
Não são sites exclusivamente de vagas de front, mas você encontra:

- [Hipsters Jobs](https://hipsters.jobs/)
- [Cubo Jobs](https://cubo.network/jobs)
- [Apinfo](http://www.apinfo.com/)

**Comment by MatheusGoncalvs** [comment]
Um site que sempre uso é este: [Love Mondays](https://lovemondays.com.br)
